simulation/traffic_simulator/include/traffic_simulator/math/catmull_rom_spline.hpp

```diff
@@ -39,9 +39,8 @@ class CatmullRomSpline
   const std::vector<geometry_msgs::msg::Point> getTrajectory(
     double start_s, double end_s, double resolution) const;
   boost::optional<double> getSValue(
-    geometry_msgs::msg::Point position, double threshold_distance = 3.0,
-    unsigned int initial_resolution = 30, unsigned int max_iteration = 30,
-    double tolerance = 0.001);
+    geometry_msgs::msg::Point position, unsigned int initial_resolution = 30,
+    unsigned int max_iteration = 30, double tolerance = 0.03);
```

Comment:
`unsigned int` => `std::size_t`

https://stackoverflow.com/questions/131803/unsigned-int-vs-size-t

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/603/commits/71e2f34e3d6f0fa135e9708cba12d90aa0421824